server.js

```diff
@@ -1,15 +1,27 @@
 'use strict';
-var crypto = require('crypto');
-var Q = require('q');
+
+/* Libraries */
+var compression = require('compression');
 var connect = require('connect');
-var morgan = require('morgan');
+var crypto = require('crypto');
 var errorhandler = require('errorhandler');
+var morgan = require('morgan');
+var yeomanPluginCache = require('yeoman-plugin-cache');
 var timeout = require('connect-timeout');
-var compression = require('compression');
-var pluginCache = require('./');
 
-var HTTP_PORT = process.env.PORT || 8001;
 
+/* env Variables */
```

Comment:
I didn't want to add a library just to use extend. It wouldn't really save much or add much value here.